### ISSUE TITLE
How to initialize the regressor and classifier?

### ISSUE THREAD

**Issue by HsLOL** [zylo117/Yet-Another-EfficientDet-Pytorch#633]
Hi @zylo117 
I want to intialize the regressor and classifier. And I use the following code to realize it. But I print the parameters of the regressor and classifier, I think I am wrong.

    if opt.init_head:
        def init_head(m):
            classname = m.__class__.__name__
            for ntl in ['regressor', 'classifier']:
                if ntl in classname:
                    if isinstance(m, nn.Conv2d):
                        nn.init.constant_(m.weight.data, 1)
                        nn.init.constant_(m.bias.data, 0.5)
                    elif isinstance(m, nn.BatchNorm2d):
                        nn.init.constant_(m.weight, 1)
                        nn.init.constant_(m.bias, 0)
        model.apply(init_head)
        print('[Info] init head(regressor & classifier)')

Could you tell me how to intialize the parameters of the regressor and classifier?
Thanks a lot!

**Comment by zylo117** [comment]
there is a init weight function somewhere in this repo. search it
